modules/serverless-install-kafka-odc.adoc

```diff
@@ -0,0 +1,34 @@
+// Module is included in the following assemblies:
+//
+// serverless/serverless-kafka.adoc
+
+[id="serverless-install-kafka-odc_{context}"]
+= Installing Apache Kafka components using the web console
+
+Cluster administrators can enable the use of Apache Kafka functionality in a {ServerlessProductName} deployment by instantiating the Kafka custom resource definition provided by the *Knative Kafka* {ServerlessOperatorName} API.
```

Comment:
maybe "the KnativeKafka custom resource definition"?

(not sure if the word "Kafka" is here meant as literally the "kind" of the CRD, which is KnativeKafka  ( knativekafkas.operator.serverless.openshift.io  ) 

eg. the user can list them by

oc get KnativeKafka --all-namespaces

Doing "oc get Kafka --all-namespaces"  would return the instances of kafkas.kafka.strimzi.io instead)